packages/page-statistics/src/provider/hooks/useSession.ts

```diff
@@ -0,0 +1,83 @@
+import { useLatestThresholdsLazyQuery } from '@webb-dapp/page-statistics/generated/graphql';
+import { thresholdMap } from '@webb-dapp/page-statistics/provider/hooks/mappers/thresholds';
+import { Loadable } from '@webb-dapp/page-statistics/provider/hooks/types';
+import { useStatsContext } from '@webb-dapp/page-statistics/provider/stats-provider';
+import { useEffect, useState } from 'react';
+
+type SessionThresholdValue = {
+  sessionId: string;
+  keygenThreshold: number;
+  signatureThreshold: number;
+};
+type LatestThresholdsValue = Loadable<Array<SessionThresholdValue>>;
+
```

Comment:
Can we add a comment here about the purpose of the `isLatest` flag?

Reply:
Sure